tests/test_data/matsim/network.xml

```diff
@@ -2,7 +2,7 @@
 <!DOCTYPE network SYSTEM "http://www.matsim.org/files/dtd/network_v2.dtd">
 <network>
 	<attributes>
-		<attribute name="crs" class="java.lang.String">{'init': 'epsg:27700'}</attribute>
+		<attribute name="crs" class="java.lang.String">epsg:27700</attribute>
```

Comment:
Wasn't sure of the implications of having to make change to a test network - could this mean existing networks will be broken now? I used the branch of genet for network simplification in a Matesto smoketest pipeline, and nothing fell over, but that doesn't feel like comprehensive reassurance. Presumably any network written by an older version of GeNet (or Puma?) will include the CRS attribute in the deprecated style?

Reply:
I don't think this will break anything actually, lol
GeNet just writes the projection to xml, doesnt use it (user has to specify the projection for the network always), it does use simplified tag though https://github.com/arup-group/genet/blob/909e03422c673db83e987cf7fcaeb0632c435078/genet/inputs_handler/read.py#L47  which gets read from xml here: https://github.com/arup-group/genet/blob/909e03422c673db83e987cf7fcaeb0632c435078/genet/inputs_handler/matsim_reader.py#L161 

I think it would be cool to try read the projection from xml, but we've been doing fine with people specifying each time so I don't feel a strong pull towards it

Reply:
Ah, okay. Yes, reading the projection from XML seems like a nice-to-have at best.